pkg/block/index.go

```diff
@@ -14,17 +14,20 @@ import (
 	"strings"
 	"time"
 
+	"github.com/prometheus/prometheus/storage"
+
 	"github.com/thanos-io/thanos/pkg/block/metadata"
```

Comment:
Can you please help group these imports?

Reply:
I've fixed grouping of imports in all modified files, using common grouping order: stdlib, 3rd-party imports, and project imports. It may be good to add linter for this (eg. Cortex has one), but that would be a separate PR imho.